Ginger/GingerCore/Environments/Database.cs

```diff
@@ -394,7 +394,10 @@ public Boolean Connect(bool displayErrorPopup = false)
 
                     case eDBTypes.Cassandra:
                         GingerCassandra CassandraDriver = new GingerCassandra(this);
-                        CassandraDriver.Connect();
+                        bool cassandraConnection;
```

Comment:
cassandraConnection is more relevant if it is connection object.
As it is bool please rename to something like isConnected